src/SFA.DAS.EmployerRequestApprenticeTraining.Web/Controllers/EmployerRequestController.cs

```diff
@@ -131,17 +133,15 @@ public async Task<IActionResult> Existing(SubmitEmployerRequestParameters parame
         [ServiceFilter(typeof(ValidateRequiredQueryParametersAttribute))]
         public async Task<IActionResult> Start(SubmitEmployerRequestParameters parameters)
         {
-            await _orchestrator.StartEmployerRequest(parameters.Location);
-            return RedirectToRoute(EnterApprenticesRouteGet, new { parameters.HashedAccountId, parameters.RequestType, parameters.StandardId, parameters.Location, BackToCheckAnswers=false });
+            return RedirectToRoute(EnterApprenticesRouteGet, new { parameters.HashedAccountId, BackToCheckAnswers=false });
         }
 
         [HttpGet]
         [Route("cancel", Name = CancelEmployerRequestRouteGet)]
         [ServiceFilter(typeof(ValidateRequiredQueryParametersAttribute))]
-        public async Task<IActionResult> Cancel(SubmitEmployerRequestParameters parameters)
+        public async Task<IActionResult> Cancel(OverviewParameters parameters)
```

Comment:
This should take a SubmitEmployerRequestParameters as it doesn't use any of the new properties in the OverviewParameters model

Reply:
updated